truss/templates/base.Dockerfile.jinja

```diff
@@ -1,4 +1,5 @@
 ARG PYVERSION={{ config.python_version }}
+ARG HOME
```

Comment:
Wouldn't this rely on a build arg that we don't set to be passed? I'm not too familiar with Dockerfiles, but it seems like this will get ignored? 

After some brief testing it seems like we'll ignore this and effectively fall back to the HOME in the base image, but am I missing when this is useful?